### PR TITLE
fix: version numbers (#138)

### LINKED ISSUE #138: read versionnumber directly from env
we could use `process.env.npm_package_version` instead of reading the version-number with the config-serivce and the env-variables like [here](https://github.com/EddieHubCommunity/api/blob/9813ac19d735da63fa93ceb36e0ffa74b3a76031/src/app.service.ts#L10).

### PR DESCRIPTION
**Pull request by eddiejaoude**
closes #138

### CODE REVIEW

File: test/features/hello-world.feature [hunk 1]
```diff
@@ -4,4 +4,4 @@ Feature: hello world
     Scenario: welcome message
         When make a GET request to "/"
         Then the response status code should be 200
-        And the response should be "Welcome to EddieHub! Currently running version: v0.0.0"
+        # And the response should be "Welcome to EddieHub! Currently running version: v0.0.0"
```
Comment (eddiejaoude) [inline test/features/hello-world.feature:7]: We need an issue for this! Would anyone like a green square 🟩

Comment (SashoStoichkov) [reply test/features/hello-world.feature:7]: Issue: #141

Comment (eddiejaoude) [reply test/features/hello-world.feature:7]: Issue raised here #141

**Review by nhcarrigan — APPROVED** [review]

**Review by SashoStoichkov — APPROVED** [review]
Looks great :) Amazing collaboration @eddiejaoude @Cahllagerfeld

Done on stream: https://www.youtube.com/watch?v=F6mLySx9Kzc

**Pull request merged**